app/styles/app.scss

```diff
@@ -218,7 +218,7 @@ article.aria-roles {
     margin-bottom: 0.5em;
   }
 .demo-div {
-  flex: 1 1 1;
```

Comment:
Wow flexbox support is so much further along than I thought.

I personally wouldn't worry about IE9 as long as it doesn't pose problems for AT user agents.

![image](https://cloud.githubusercontent.com/assets/831043/15401416/5ef4fbe8-1da4-11e6-9017-19282bcacfd7.png)